lib/resque/cluster/config.rb

```diff
@@ -7,17 +7,24 @@ class Cluster
     class Config
       extend Forwardable
 
-      attr_reader :configs, :local_config, :global_config, :verifier, :version_git_hash
+      attr_reader :configs, :config, :global_config, :verifier, :version_git_hash
```

Comment:
config as a variable name makes this even more confusing, could we call it something like universal_config or combined_config